credentialz/gnsi-credentialz.yang

```diff
@@ -22,6 +22,12 @@ module gnsi-credentialz {
         "This module provides a data model for the metadata of SSH and console
          credentials installed on a networking device.";
 
+    revision 2023-08-18 {
```

Comment:
You can probably just use a patch revision here if you'd prefer, but this is also fine.

Reply:
Sorry, nonblocking.